spec/frameSpec.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
```

Comment:
Good testing unit testing of the logic - all 15 tests written are passing. As you mentioned in your README, would have also be good to have some feature testing if you had more time.